### ISSUE TITLE
Error adding watch for files

### ISSUE THREAD

**Issue by garvincasimir** [GoogleContainerTools/skaffold#287]
### Expected behavior
Watch should be added for Dockerfile and other deps
### Actual behavior
Error:
adding watch for C:\Users\garvin\projects\kube-task\job\Dockerfile: The parameter is incorrect

### Information
I looked at the code and the error seems to be coming from here:
https://github.com/GoogleCloudPlatform/skaffold/blob/739a5048151dc610956fa7117075add73bff1795/pkg/skaffold/watch/watch.go#L202

Perhaps this is an issue with http://github.com/rjeczalik/notify and Windows?

Skaffold version: v0.2.0
Operating system: Windows 10 pro (build 16299)
Content of skaffold.yaml: 
```
apiVersion: skaffold/v1alpha1
kind: Config
build:
  artifacts:
  - imageName: localhost:5000/kube-task 
    workspace: C:\Users\garvin\projects\kube-task\job
  local: {}
deploy:
  kubectl:
    manifests:
    - paths:
      - k8s/*
      parameters:
        KUBE-TASK: localhost:5000/kube-task
```
### Steps to reproduce the behavior

  1. Run skaffold dev
  2. Error appears after pod deployment

**Comment by jguo13** [comment]
I also had the same issue trying to run skaffold on windows 7 after running skaffold dev, is there a fix for this?

**Comment by Anakin-Hao** [comment]
Running into the same issue.
My Env:
Win 10 Pro
skaffold 0.3.0

**Comment by F21** [comment]
I also ran into the same problem with Windows 10 (latest version) and skaffold 0.3.0.

However, after building master (7349a0f3bc97926c5a894bb1e0af7c63e254a373), this is no longer happens.

To use master, install Go, delete the old skaffold binary and run `go get -u github.com/GoogleCloudPlatform/skaffold/cmd/skaffold`. Assuming `$GOPATH/bin` is in your PATH, you can run `skaffold` from anywhere.

I believe this might be due to switching to fsnotify a few days ago.

**Comment by dgageot** [comment]
Fixed with #620
